frontend/src/modules/cruise-applications/pages/ApplicationsPage.tsx

```diff
@@ -1,68 +1,131 @@
 import { ColumnDef } from '@tanstack/react-table';
 import { Suspense } from 'react';
+import ZoomInIcon from 'bootstrap-icons/icons/zoom-in.svg?react';
 
 import { AppLayout } from '@/core/components/AppLayout';
 import { AppLink } from '@/core/components/AppLink';
 import { AppLoader } from '@/core/components/AppLoader';
 import { AppTable } from '@/core/components/table/AppTable';
 import { useCruiseApplicationsQuery } from '@/cruise-applications/hooks/CruiseApplicationsApiHooks';
-import { CruiseApplicationDto } from '@/cruise-applications/models/CruiseApplicationDto';
+import { CruiseApplicationDto, CruiseApplicationStatus } from '@/cruise-applications/models/CruiseApplicationDto';
+import { AppAvatar } from '@/core/components/AppAvatar';
+import { AppButton } from '@/core/components/AppButton';
+import { AppBadge } from '@/core/components/AppBadge';
 
 export function ApplicationsPage() {
   const applicationsQuery = useCruiseApplicationsQuery();
 
   const columns: ColumnDef<CruiseApplicationDto>[] = [
     {
-      header: 'Numer/data',
-      accessorFn: (row) => `${row.number}/${row.date}`,
+      id: 'number',
+      header: 'Numer',
+      accessorFn: (row) => `${row.number}`,
```

Comment:
AFAIK it is not required that accessorFn returns string, so it could be refactored to `(row) => row.number`.

Apply to others as well.